toolbox/fdc3-conformance/Intents-Tests.md

```diff
@@ -0,0 +1,247 @@
+  - Then in the first step, use `fdc3.raiseIntent("aTestingIntent", testContextX, appIdentifier)` to target the running instance of app A. 
+  - Confirm that the intent is delivered to the correct instance and that another instance is NOT started. Otherwise, as above.
+- `2.0-RaiseIntentTargetedInstanceResolveFindInstances`: Repeat the above test, but:
+  - Before the first step, use `let appIdentifier = await fdc3.open({appId: "<A's appId>"})` to start A.
+  - Then use `const instances = await fdc3.findInstances({appId: "<A's appId>"})` to retrieve a list of instances of app A. Confirm that only one is present and retrieve its `AppIdentifier`, confirming that it contains an `instanceId` field that matches that returned by the `fdc3.open` call.
+  - Then in the first step, use `fdc3.raiseIntent("aTestingIntent", testContextX, appIdentifier)` to target the running instance of app A. 
+  - Confirm that the intent is delivered to the correct instance and that another instance is NOT started. Otherwise, as above.
+- `2.0-RaiseIntentFailedResolve`: Perform above test, but:
+  - Use `fdc3.raiseIntent("aTestingIntent", testContextY)`.  Note that no app supports this intent and context combination.**
+  - You should receive a JavaScript Error with the message `ResolveError.NoAppsFound`.
+- `2.0-RaiseIntentFailTargetedAppResolve1`: Perform above test, but:
+  - Use `fdc3.raiseIntent("aTestingIntent", testContextY, {appId: "<A's appId>"})`.
+  - You should receive a JavaScript Error with the message `ResolveError.IntentDeliveryFailed`.
+- `2.0-RaiseIntentFailTargetedAppResolve2`: Perform above test, but:
+  - Use `fdc3.raiseIntent("aTestingIntent", testContextX, {appId: "NonExistentApp"})`.
+  - You should receive a JavaScript Error with the message `ResolveError.TargetAppUnavailable`.
+- `2.0-RaiseIntentFailTargetedAppResolve3`: Perform above test, but:
+  - Use `fdc3.raiseIntent("sharedTestingIntent2", testContextY, {appId: "<H's appId>"})`.
+  - You should receive a JavaScript Error with the message `ResolveError.IntentDeliveryFailed` (as this app is configured for the intent and context pair, but does not add any intent listeners).
+  - **Note:  Test will need an extended timeout to allow for this to be returned in time by the desktop agent, which will have a vendor-defined timeout.**
+- `2.0-RaiseIntentFailTargetedAppResolve4`: Perform above test, but: 
+  - `fdc3.raiseIntent("sharedTestingIntent2", testContextY, {appId: "<I's appId>"})`
+  - You should receive a JavaScript Error with the message `ResolveError.IntentDeliveryFailed` (as this app is configured for the intent and context pair, but adds intent listeners of the wrong type.
+  - **Note:  Test will need an extended timeout to allow for this to be returned in time by the desktop agent, which will have a vendor-defined timeout.**
+- `2.0-RaiseIntentFailTargetedAppInstanceResolve1`: Perform above test, but:
```

Comment:
Specifications is asking to check NoAppsFound but coding is testing 'IntentDeliveryFailed' and it's passing.

Reply:
I think thats a bug on our end and we should be changing to return NoAppsFaound - will check and confirm

Reply:
I think IntentDeliveryFailed should be reserved for cases where you should be able to deliver it (as it matches config) but can't (as the listener wasn't added).